### PR TITLE
Update README.md contributor and release docs

### PR DESCRIPTION
**Pull request by YesCT**
Ready for review/merge!

Todo:
- [x] verify correct information
- [x] identify missing information
- [x] refine wording
- [x] run TOC generation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -547,3 +560,33 @@ tugboat:php:init:
...
     - docker-php-ext-install ldap
 ```
+
+## Contributor Docs
+
+This repo is public.
+
+Please be careful to remove sensitive customer specifics when posting Issues or comments.
+
+Firsttime contributors need a maintainers approval for automated tests to run.
```
Comment (deviantintegral) [inline README.md:570]:
```suggestion
First time contributors need a maintainers approval for automated tests to run.
```

**Comment by deviantintegral** [comment]
I've marked this as high as getting docs for the release process will eventually let us get to a scheduled release regardless of PTO or other time.

**Comment by YesCT** [comment]
I thought drainpipe-dev was supposed to have a release for every drainpipe release.

https://github.com/Lullabot/drainpipe

3.4.1

Screenshot 2024-02-20 at 9 50 30 AM

https://github.com/Lullabot/drainpipe-dev

3.40

Screenshot 2024-02-20 at 9 50 35 AM

DOES drainpipe-dev and drainpipe always need to be released together with matching versions?? Did something change? Am I mis-remembering? cc @beto-aveiga @justafish

**Comment by davereid** [comment]
> DOES drainpipe-dev and drainpipe always need to be released together with matching versions?? Did something change? Am I mis-remembering?

Yes I believe this is the case, they should always be released together with the same version. I can add some notes I took during last week's release process.

**Comment by YesCT** [comment]
@davereid I resolved the conflicts and updated the TOC. 

Did you want to ... 
>  add some notes I took during last week's release process.
before this is merged, or as a follow-up?

File: README.md [hunk 2]
```diff
@@ -602,6 +615,37 @@ tugboat:php:init:
     - docker-php-ext-install ldap
 ```
 
+<<<<<<< HEAD
```
Comment (davereid) [inline README.md:618]: Conflict remains here.

**Comment by davereid** [comment]
@YesCT I added my release process notes and solved the Git conflicts.

**Comment by YesCT** [comment]
Thank you! sorry I left the conflict partially in still.

**Review by YesCT — COMMENTED** [review]
This looks great to me! (I can't approve it, since I was the original author of the PR.)

File: README.md [hunk 3]
```diff
@@ -629,13 +649,24 @@ php:
...
-### Overview
+Peer Reviewing by looking at PR code changes is nice.
+
+Testing PR code changes on real sites is extra beneficial.
+
+### Peer Review Guidelines for Automated Updates
+
+These are guidelines for conducting peer reviews on automated dependency update pull requests created by Renovate.
+
+#### Overview
```
Comment (davereid) [inline README.md:669]: Do we need an overview section here? Could it not just be a part of this section above with only one sentence?

Comment (davereid) [reply README.md:669]: cc @mrdavidburns just trying to figure out the best way to merge this new section with the work that was merged in https://github.com/Lullabot/drainpipe/pull/411

**Review by davereid — APPROVED** [review]

File: .gitignore [hunk 4]
```diff
@@ -13,3 +13,10 @@ tests/fixtures/drainpipe-test-project/*
 !.yarn/sdks
 !.yarn/versions
 .npmrc
+
+################
+# Editors / OS #
+################
+.idea
+.DS_Store
+.vscode
```
Comment (deviantintegral) [inline .gitignore:23]: @YesCT you might be interested in https://github.com/github/gitignore/tree/main?tab=readme-ov-file. It's generally recommended to put these in your global `.gitignore` so it automatically applies to all projects.

Comment (YesCT) [reply .gitignore:23]: Will do. And I'll pass on that feedback to our other projects where we add this.

**Pull request merged**